pom.xml

```diff
@@ -469,6 +480,31 @@
           <useSystemClassLoader>false</useSystemClassLoader>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>2.0.0</version>
+        <executions>
+          <execution>
+            <configuration>
+              <verbose>false</verbose>
+              <thirdPartyFilename>THIRD-PARTY-Licenses.txt</thirdPartyFilename>
```

Comment:
Why is the default name `THIRD-PARTY.txt` being changed to `THIRD-PARTY-Licenses.txt`? This will make it more difficult to collect all licenses of a complete FOLIO installation ( https://github.com/folio-org/platform-complete ).